### PR TITLE
Limit transcripts assistant picker to custom assistants only

### PR DESCRIPTION
**Pull request by albandum**
## Description

For now people could add @gpt4 and all default assistants as transcripts processors, which makes no sense. 
Removing the possibility to use a default assistant in Transcripts

### CODE REVIEW

File: front/pages/w/[wId]/assistant/labs/transcripts/index.tsx [hunk 1]
```diff
@@ -130,7 +130,10 @@ export default function LabsTranscriptsIndex({
     return <Spinner />;
   }
 
-  const agents = agentConfigurations.filter((a) => a.status === "active");
+  // Only custom assistants - there's no point in passing a transcript to a default assistant without instructions.
+  const agents = agentConfigurations.filter(
+    (a) => a.status === "active" && a.id != -1
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/labs/transcripts/index.tsx:135]: Don't we have a function to filter them out?

Comment (spolu) [reply front/pages/w/[wId]/assistant/labs/transcripts/index.tsx:135]: You likely can tweak that instead: agentsGetView: "list",

Comment (albandum) [reply front/pages/w/[wId]/assistant/labs/transcripts/index.tsx:135]: Are we actually using agentsGetView? 

From the SWR `useAgentConfigurations` it looks like it, but looking at `front/pages/api/v1/w/[wId]/assistant/agent_configurations.ts` the GET request does not take into account any query param :/

```
const agentConfigurations = await getAgentConfigurations({
        auth,
        agentsGetView: "all",
        variant: "light",
      });
```

Comment (spolu) [reply front/pages/w/[wId]/assistant/labs/transcripts/index.tsx:135]: oh it should cc @philipperolet who introduced it I believe?

**Pull request closed without merging**